tap_bing_ads/reports.py

```diff
@@ -32,7 +32,13 @@
         'Ctr',
         'Impressions'
     ],
-    'GoalsAndFunnelsReport': ['Goal']
+    'GoalsAndFunnelsReport': ['Goal'],
+    'AgeGenderAudienceReport': [
```

Comment:
@namrata270998 Can you please provide a comment above this code change?

Reply:
Added comments